updater/scripts/api-requests-by-user.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+#
+# Count number of API calls per user
+#
+
+echo -e "user\trequests"
+
+zcat -f /var/log/github/unicorn.log.1* |
+    grep -F 'request_category=api' |
+    grep -Fv 'remote_address=127.0.0.1' |
```

Comment:
@dfarr I excluded internal API calls here as they are not directly issued by a user (e.g. Git LFS requests)

Reply:
Good idea 👍